Makefile

```diff
@@ -124,12 +124,19 @@ run-dev: chronogiraffe
 	mkdir -p ui/build
 	./chronograf -d --log-level=debug
 
+e2e-prepare:
+	./ui/cypress/local-chronograf-influxdb-enterprise.sh
+
+e2e:
+	cd ui && yarn test:e2e
+
 clean:
 	if [ -f ${BINARY} ] ; then rm ${BINARY} ; fi
 	cd ui && yarn run clean
 	cd ui && rm -rf node_modules
 	rm -f dist/dist_gen.go canned/bin_gen.go protoboards/bin_gen.go server/swagger_gen.go
 	@rm -f .godep .jsdep .jssrc .bindata
+	./ui/cypress/local-chronograf-influxdb-enterprise.sh 0 1
```

Comment:
@tomklapka the clean target is run by circle CI (indirectly via a builder docker, in python) and it will fail. We should be rather deleting the files herein only, it seems to me that we are going to delete a cluster using `kind` might not even be installed ... we should rather find a better path of doing cleanups before running e2e tests.

Reply:
I can remove it if you feel that it won't work.

Reply:
I see that it will not fail in the CI, it only silently executes binaries that do not exist. I don't know why this cleanup is necessary herein,  CI IMHO does not need it and a local installation can have an extra target named e2e-clean (if required) ... anyway, the executed script performs cleanup every time it starts.

Reply:
done